pkg/job/task.go

```diff
@@ -256,6 +265,7 @@ func (r *TaskRunner) Run() (types.TaskStatus, error) {
 		r.tryRunAssert,
 		r.tryRunDelete,
 		r.tryRunApply,
+		r.tryRunDeleteAll,
```

Comment:
can you rearrange `tryRunDeleteAll` just after `tryRunDelete`

Reply:
Resolved this comment